dev/AnimatedIcon/AnimatedVisuals/Controls_01_Checkmark.idl

```diff
@@ -18,12 +18,13 @@
 //     the code is regenerated.
 // </auto-generated>
 //------------------------------------------------------------------------------
+//------------------------------------------------------------------------------
 namespace MU_XCAV_NAMESPACE
 {
     [WUXC_VERSION_PREVIEW]
     [webhosthidden]
     runtimeclass Controls_01_Checkmark
-        : [default] Microsoft.UI.Xaml.Controls.IAnimatedVisualSource, Microsoft.UI.Xaml.Controls.IRichAnimatedVisualSource
+        : [default] Microsoft.UI.Xaml.Controls.IAnimatedVisualSource, Microsoft.UI.Xaml.Controls.IAnimatedVisualSource2
     {
         Controls_01_Checkmark();
```

Comment:
nit: I assume these names will change

Reply:
Yes, but I want to wait till I get approved names..

---
In reply to: [580411143](https://github.com/microsoft/microsoft-ui-xaml/pull/4277#discussion_r580411143) [](ancestors = 580411143)